### PR TITLE
MP-5146 Actions Drawer

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/MP-5146

### CODE REVIEW

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  | DrawerActionConfigComponent
+  | DrawerActionConfigTemplate;
+
+// Component registration
+@NgModule({
+  imports: [
+    ActionsModule.withActions({
+      drawer: DrawerActionHandlerService,
+    }),
+    DrawerActionModule.withComponents({
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:58]: Please provide `component registration` section with describing the component registration.
And remove this from interfaces.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+```ts
+handleAction(injector, config: { component: SimpleComponent }, context);
+```
+
+Then `DrawerRef.openComponent` method will be called without any manipulations with `DrawerActionComponentTypesToken`.
+
+### Via Template
+
+There is an easy way to open drawer with `ng-template`. All you need it's just create a template and call
```
Comment (supproduction) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:76]:
```suggestion
Another way to open drawer with `ng-template`. You need create a template and call 
```

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+```html
+  <ng-template #contentTpl>
+     ...
+  </ng-template>
+```
+
+```ts
+// Find the template
+@ViewChild(‘contentTpl’) contentTpl?: TemplateRef<any>;
```
Comment (supproduction) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:87]:
```suggestion
import {  DrawerTemplateContext } from '@spryker/drawer';

@ViewChild(‘contentTpl’) contentTpl?: TemplateRef<DrawerTemplateContext>;
```

**Review by supproduction — CHANGES_REQUESTED** [review]

**Review by supproduction — DISMISSED** [review]

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+title: Actions Drawer
+description: This article provides details about the Actions Drawer service in the Components Library.
+template: concept-topic-template
+---
+
+This article provides details about the Actions Drawer service in the Components Library.
+
+## Overview
+
+Actions Drawer is an Angular Service that opens component/template in the Drawer.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:11]:
```suggestion
Actions Drawer is an Angular Service that opens the Drawer with a component/template.
```

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+```ts
+handleAction(injector, config: { component: SimpleComponent }, context);
+```
+
+Then `DrawerRef.openComponent` method will be called without any manipulations with `DrawerActionComponentTypesToken`.
+
+### Via Template
+
+Another way to open drawer with `ng-template`. You need to create a template, get reference 
+of it and throw it as `template` config prop to the `handleAction` method.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:77]:
```suggestion
Another way to open the Drawer with `ng-template`. You need to create a template, get the reference of it and throw it as `template` config prop to the `handleAction` method.
```

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+// Find the template
+import { DrawerTemplateContext } from '@spryker/drawer';
+
+@ViewChild(‘contentTpl’) contentTpl?: TemplateRef<DrawerTemplateContext>;
+
+// Call the method 
+handleAction(injector, config: { template: contentTpl }, context);
+```
+
+`DrawerRef.openTemplate` will be called and drawer will be open with `contentTpl` template.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:95]:
```suggestion
`DrawerRef.openTemplate` will be called and the Drawer will be open with `contentTpl` template.
```

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+## Service registration
+
+Any existing Angular component can be registered and used within the drawer.
+Also, it's possible to create and register a custom component that will be rendered inside the drawer.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-drawer.md:100]:
```suggestion
Any existing Angular component can be registered and used within the Drawer.
Also, it's possible to create and register a custom component that will be rendered inside the Drawer.
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**